code/datums/components/tackle.dm

```diff
@@ -0,0 +1,485 @@
+				T.add_movespeed_modifier(MOVESPEED_ID_SHOVE, multiplicative_slowdown = SHOVE_SLOWDOWN_STRENGTH) // maybe define a slightly more severe/longer slowdown for this
+				addtimer(CALLBACK(T, /mob/living/carbon/human/proc/clear_shove_slowdown), SHOVE_SLOWDOWN_LENGTH)
+
+		if(-1 to 0) // decent hit, both parties are about equally inconvenienced
+			user.visible_message("<span class='warning'>[user] lands a passable tackle on [target], sending them both tumbling!</span>", "<span class='userdanger'>You land a passable tackle on [target], sending you both tumbling!</span>", target)
+			to_chat(target, "<span class='userdanger'>[user] lands a passable tackle on you, sending you both tumbling!</span>")
+
+			target.adjustStaminaLoss(stamina_cost)
+			target.Paralyze(5)
+			user.Knockdown(20)
+			target.Knockdown(25)
+
+		if(1 to 2) // solid hit, tackler has a slight advantage
+			user.visible_message("<span class='warning'>[user] lands a solid tackle on [target], knocking them both down hard!</span>", "<span class='userdanger'>You land a solid tackle on [target], knocking you both down hard!</span>", target)
+			to_chat(target, "<span class='userdanger'>[user] lands a solid tackle on you, knocking you both down hard!</span>")
+
+			target.adjustStaminaLoss(30)
+			target.Paralyze(5)
+			user.Knockdown(10)
+			target.Knockdown(20)
+
+		if(3 to 4) // really good hit, the target is definitely worse off here. Without positive modifiers, this is as good a tackle as you can land
+			user.visible_message("<span class='warning'>[user] lands an expert tackle on [target], knocking [target.p_them()] down hard while landing on [user.p_their()] feet with a passive grip!</span>", "<span class='userdanger'>You land an expert tackle on [target], knocking [target.p_them()] down hard while landing on your feet with a passive grip!</span>", target)
+			to_chat(target, "<span class='userdanger'>[user] lands an expert tackle on you, knocking you down hard and maintaining a passive grab!</span>")
+
+			user.SetKnockdown(0)
+			user.forceMove(get_turf(target))
+			target.adjustStaminaLoss(40)
+			target.Paralyze(5)
+			target.Knockdown(30)
+			if(ishuman(target) && ishuman(user))
+				S.dna.species.grab(S, T)
+				S.setGrabState(GRAB_PASSIVE)
+
+		if(5 to INFINITY) // absolutely BODIED
+			user.visible_message("<span class='warning'>[user] lands a monster tackle on [target], knocking [target.p_them()] senseless and applying an aggressive pin!</span>", "<span class='userdanger'>You land a monster tackle on [target], knocking [target.p_them()] senseless and applying an aggressive pin!</span>", target)
+			to_chat(target, "<span class='userdanger'>[user] lands a monster tackle on you, knocking you senseless and aggressively pinning you!</span>")
+
+			user.SetKnockdown(0)
+			user.forceMove(get_turf(target))
+			target.adjustStaminaLoss(40)
+			target.Paralyze(5)
+			target.Knockdown(30)
+			if(ishuman(target) && ishuman(user))
+				S.dna.species.grab(S, T)
```

Comment:
since you can execute tackles while cuffed, what happens if you end up rolling this result (or the 3 to 4 result) while cuffed?

Reply:
yeah i just removed this